app/models/agents/data_output_agent.rb

```diff
@@ -165,6 +166,10 @@ def feed_description
       interpolated['template']['description'].presence || "A feed of Events received by the '#{name}' Huginn Agent"
     end
 
+    def rss_content_type
+      interpolated['rss_content_type'] || 'application/rss+xml'
```

Comment:
maybe `interpolated['rss_content_type'].presence || 'application/rss+xml'` to ignore blanks?